orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -69,7 +71,7 @@ def is_disabled(self):
 
 
```

Comment:
Some functions with no return statements will have -> None as a return type but some won't, how do we decide which ones to have -> None and which ones to not have it ?

example : def _set_device_from_module(self, inputs, kwargs):